python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -15,7 +15,7 @@
 logger = getLogger()
 
 _reg_id_str = '_registry_id'
-_id_str = 'id'
+_id_str = '_id'
```

Comment:
1. `_id_str` doesn't seem to be used anywhere
2. I don't like the idea of an object being told what its registry id (`_registry_id`) is. The registry should know but apart from that it should be internal to the registry. Too much coupling in both directions between the registry and the objects within is part of what's causing a lot of the lockup problems. If an object wants to know what its id is then it should query its registry to find out. Duplicating this information is likely just increasing the chance of them getting out of sync.